### PR TITLE
Capstone Project One

### PR DESCRIPTION
**Pull request by MrBrN197**
### Capstone One

- Set up the eslint, webhint, stylelint.
- Setup HTML, CSS, sass files
- Created about and home page.
- Deployed the project
- Create a desktop version for about and home page.
- Implemented mobile menu modal.
- Implemented the featured section with dynamic HTML.

## Video Presentation Demo Link
https://www.loom.com/share/554b31c2303b413fb0465a7f14fc66fa
- In the video, I demonstrate using the desktop and mobile responsive design of the website
- the links that navigate to different parts of the page.

## Code for More-Button and Mobile Menu Feature Demo Link
https://www.loom.com/share/d7e8290bc86d4bdc9f7255a294301ea5
- the "MORE" but is set to show more featured artists when you click on it. 
- the "MORE" icon is set to a loading SVG when clicked is remove from the page when all featured artists are displayed
- the mobile menu that scrolls from the side using javascript and CSS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+> In this is a website designed using css, javascript and html based on this design <https://www.behance.net/gallery/29845175/CC-Global-Summit-2015>
+
+## Built With
+
+- Eslint
+- Webhint
+- Styelint
+- CSS/Sass
+
+## Live Demo
```
Comment (carlos-ssh) [inline README.md:16]: Kindly work on these requested changes:
**- Provide feedback about the video presentation in the code review summary.** 

1. The maximum length of the video is 5 minutes. 
2. The video contains a demo of the project, showing the pages in different screen sizes. 
3. Provide a highlight about a piece of the code the student finds interesting or is proud of making.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (carlos-ssh) [inline about.html:3]: - Your project it's missing the About page.

1. The project contains a Home page and About page.
2. It's possible to navigate between the pages, there's a link from Home page to About page and the opposite too.
3. Both pages contain the same navbar and footer.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+  <link href="fonts/cocogoose/cocogoose.css" rel="stylesheet">
+  <link rel="stylesheet" href="./css/global.css">
+  <link rel="stylesheet" href="./css/about.css">
+  <script src="./script.js"></script>
+</head>
+
+<body>
+  <div id="mobile-menu">
```
Comment (carlos-ssh) [inline about.html:27]: **Mobile Size**
The hamburger menu on the left side: 
1. It's background is transparent or the same color/pattern as the top section background.
2. It's missing functionality.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+    <ul>
+      <li><a href="/">Home</a></li>
+      <li><a href="/about.html">About</a></li>
+      <li><a href="/about.html">Program</a></li>
+      <li><a href="/about.html">Sponsor</a></li>
+      <li><a href="/about.html">News</a></li>
+    </ul>
+  </div>
+  <header>
+    <nav>
```
Comment (carlos-ssh) [inline about.html:54]: **Desktop Size**
- Needs to have these:
1. It is divided in a top dark bar and a bottom white bar.
2. The top short bar section with social media links, logout, and language links aligned to the right.
3. The taller bottom bar with a logo and a list of page links.
4. The top bar contains a logo to the left.
5. The top bar contains a list of page links aligned to the right.
Screen Shot 2021-09-07 at 18 58 49

**Yours:**
Screen Shot 2021-09-07 at 18 59 28

File: about.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+      <ul class="flex flex-wrap flex-center">
+        <li><img src="./images/icons/mozilla.svg" alt="mozilla-partner"></li>
+        <li><img src="https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png" alt="google-partner"></li>
+        <li><img src="./images/icons/naver.svg" alt="naver-partner"></li>
+        <li><img src="https://media-prd.coachella.com/templates/template10/svgs/coachella-logo.svg" alt="coachella-partner"></li>
+        <li><img src="./images/icons/airbnb.svg" alt="airbnb-partner"></li>
+      </ul>
+    </section>
+  </main>
+  <footer>
```
Comment (carlos-ssh) [inline about.html:161]: **Footer**
- It contains a logo aligned to the left and text aligned to the right.
The template:
Screen Shot 2021-09-07 at 19 01 05

**Yours:**
Screen Shot 2021-09-07 at 19 01 49

#
**Mobile Size**
- It has a white background.

**Desktop Size**
- It has a dark background.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+</head>
+
+<body>
+  <div id="mobile-menu">
+    <div class="close-btn">
+      <svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-x"><line x1="18" y1="6" x2="6" y2="18"></line><line x1="6" y1="6" x2="18" y2="18"></line></svg>
+    </div>
+    <ul>
+      <li><a href="/">Home</a></li>
+      <li><a href="/about.html">About</a></li>
```
Comment (carlos-ssh) [inline about.html]: **Requirements for your About Page:**

- It contains a top section with a centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it.
- It contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo.
- It contains a bottom section with a centered column that includes a section header, a divider, a line of text, and two images with text centered over them.

**_Kindly provide the requested style on your Template._**

File: index.html [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+</head>
+
+<body>
+  <div id="mobile-menu">
+    <div class="close-btn">
+      <svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-x"><line x1="18" y1="6" x2="6" y2="18"></line><line x1="6" y1="6" x2="18" y2="18"></line></svg>
+    </div>
+    <ul>
+      <li><a href="/about.html">About</a></li>
+      <li><a href="/about.html">Program</a></li>
```
Comment (carlos-ssh) [inline index.html]: [OPTIONAL]
- Disable unused ` <a> ` tags.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @MrBrN197 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 2 above)
Comment (MrBrN197) [reply about.html:3]: Hi, thanks for the comment. but I have both the home and about page. the home page Is index.html and the about page is about.html. the header has links to both these pages.  can you clarify some more what you mean by the missing About page?

File: about.html (hunk 3 above)
Comment (MrBrN197) [reply about.html:27]: 1. My hamburger icon is in fact transparent could you clarify more on what you mean with the first point. 
2. with the second point I'm not sure what you mean by missing functionality

File: README.md (hunk 1 above)
Comment (MrBrN197) [reply README.md:16]: I'll do this thank you.

File: index.html (hunk 7 above)
Comment (MrBrN197) [reply index.html]: I'll do this thank you.

**Comment by rloterh** [comment]
> ### Capstone One
> * Set up the eslint, webhint, stylelint.
> * Setup html, css, sass files
> * Created about and home page.
> * Deployed the project
> * Create desktop version for about and home page.
> * Implemented mobile menu modal.
> * Implemented the featured section with dynamic HTML.
> 
> ## Video Demo Link
> https://www.loom.com/share/554b31c2303b413fb0465a7f14fc66fa

- [x] Good job for adding video presentation of the project, however kindly endeavor to add in the video description some **salient aspects** of your code

File: README.md (hunk 1 above)
Comment (rloterh) [inline README.md:16]: - [x] Good job for adding video presentation of the project, however kindly endeavor to add in the video description some **salient aspects** of your code

Comment (rloterh) [reply README.md:16]: Not fully resolved.

File: README.md [hunk 8]
```diff
@@ -1,3 +1,43 @@
...
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+`git clone https://github.com/MrBrN197/Capstone-Project-One.git`
+
+## Authors
+
+👤 **Author1**
```
Comment (rloterh) [inline README.md]: - [x] OPTIONAL
Kindly replace the text `**Author1**` with your full name, so as to make your repo more professional

File: about.html [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-B899BJXW3P"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
```
Comment (rloterh) [inline about.html:11]: - [x] OPTIONAL
It is a nice idea to prefer `arrow functions ` over normal functions, so as to keep your _function_ syntax simpler and cleaner.

File: about.html (hunk 3 above)
Comment (rloterh) [reply about.html:27]: This issue is well resolved! ✔️

File: about.html [hunk 10]
```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (rloterh) [inline about.html:3]: - [x] Nice job for adding the _About_ page. However please provide a link on the _About_ page that will send visitors back to the _Home_ page when clicked. You can change the `about menu link` on your _About_ page to a `home link` within the menu.

File: about.html (hunk 4 above)
Comment (rloterh) [reply about.html:54]: This issue is well resolved! ✔️

File: about.html (hunk 5 above)
Comment (rloterh) [reply about.html:161]: This issue is well resolved! ✔️

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Orora, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦   

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments above, as well as below made under the reviews of the previous code reviewer(s) and stating **not properly resolved**. Kindly take your time to correctly review those issues outlined by the previous code reviewer(s), and please check to see that the changes are working as intended._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider the comments marked **[OPTIONAL]** under this review:

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 3 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 9 above)
Comment (MrBrN197) [reply about.html:11]: It looks like google analytics requires you to using the standard function syntax overusing arrow function so I don't think I can change this without probably causing problems with google analytics.

File: about.html (hunk 10 above)
Comment (MrBrN197) [reply about.html:3]: Thank you I've resolved this.

File: README.md (hunk 8 above)
Comment (MrBrN197) [reply README.md]: Thank You. I've changed this.

File: README.md (hunk 1 above)
Comment (MrBrN197) [reply README.md:16]: I've done the third step now. thank you

Comment (MrBrN197) [reply README.md:16]: Thank you I've added this to the description.

**Comment by Davidosky007** [comment]
## Hi @MrBrN197 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**